pkgs/shells/oksh/default.nix

```diff
@@ -11,6 +13,12 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-076nD0aPps6n5qkR3LQJ6Kn2g3mkov+/M0qSvxNLZ6o=";
   };
 
+  postPatch = lib.optionalString isCross ''
+    substituteInPlace configure --replace "./conftest" "echo"
+  '';
+
+  configureFlags = [ "--no-strip" ];
```

Comment:
What's this for?

Reply:
Without this flag `make install` will exec `install -s ...`, where `-s` - strip symbol tables. But in case of cross-compilation we have no `strip`, but some target specific strip. Anyway we don't need it, since `fixupPhase` will strip binaries for us.

Reply:
Nice